### ISSUE TITLE
Feature Request :: Query History

### ISSUE THREAD

**Issue by daniel-aranda** [wagonhq/feedback#46]
Working in a query, it would be good to have an "history" button.

![image](https://cloud.githubusercontent.com/assets/1669319/15258509/a3dd392e-1911-11e6-8e38-88fe5f2b710a.png)

So you click on it and see different SQL versions that you "ran" in that input.

Example:
- Open a new query window.
- Write

``` sql
SELECT * FROM my table
```
- Click run
- Update SQL to be:  

``` sql
SELECT COUNT(*) AS total FROM my table
```
- Click run
- Click on history button, you should see both queries listed.

**Comment by jweinstein** [comment]
Nifty, also try the history pane too. 
![image](https://cloud.githubusercontent.com/assets/547354/15258659/ae448418-1901-11e6-9630-dfa04591799f.png)

**Comment by daniel-aranda** [comment]
History panel is also cool, but scope is global. 👍

**Comment by jweinstein** [comment]
Ah, yeah we need to make this simpler. Can change to a specific query in top right or click View Previous Runs below the cards

![image](https://cloud.githubusercontent.com/assets/547354/15258767/3ac5a7fa-1902-11e6-869f-3aecd8a4bda1.png)

**Comment by daniel-aranda** [comment]
Yes, you already have the feature. Probably it just need some tweaks on UX to make it more intuitive, I was aware of them until you explained me.
